cmd/dptp-controller-manager/main.go

```diff
@@ -74,17 +78,17 @@ func main() {
 	if err != nil {
 		logrus.WithError(err).Fatal("Failed to construct manager")
 	}
+	pjutil.ServePProf()
```

Comment:
Will this pull in `interrupts` package

Reply:
Yes, why?

Reply:
If you pull it in you may need to wait on the signal there, so this does not hang when it gets SIGINT

Reply:
works on my machine